### ISSUE TITLE
Consider including test data sets

### ISSUE THREAD

**Issue by ohjakobsen** [Riksrevisjonen/pioneeR#7]
Include test data sets so that users can try out different analyses without having their own data. The Benchmarking package includes data sets that can be used for this purpose.

**Comment by ohjakobsen** [comment]
With the new `x` argument to `runPioneeR`, users can use data.sets that come with other DEA-packages directly, e.g.:
```r
runPioneeR(x = productivity::usagri)
```
However, we should still include data sets in the future, either directly in pioneeR, or through pear. I'm keeping this open in the meantime.

**Comment by JRBeckstrom** [comment]
Agree. Example data sets included for DEA noobs is a very good idea :-)

**Comment by Aeilert** [comment]
With #82 we will have *test* datasets as part of the package. These can easily be converted to example datasets available with `data()` / `pioneeR::`. We have a few datasets already documented and ready-to-go in `{pear}`, so could consider moving these to pioneeR. But depends on whether we want to wait for pear or not.
